sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -3957,3 +3957,198 @@ case class ArrayExcept(left: Expression, right: Expression) extends ArrayBinaryL
 
   override def prettyName: String = "array_except"
 }
+
+/**
+ * Checks if the array (left) has the array (right)
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(array1, array2) - Returns true if the array1 contains the array2.",
```

Comment:
can you find some examples from other databases that doing the same/similar feature?

Reply:
> can you find some examples from other databases that does the same/similar feature?

See this comment  https://github.com/apache/spark/pull/30243#issuecomment-729400441  and in our env meet this requirement.